generate/post_process_json_test.go

```diff
@@ -0,0 +1,68 @@
+package generate
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestJSONTagPostProcessor(t *testing.T) {
+	tests := []struct {
+		input  string
+		output string
+	}{
+		{
+			input: `
+package test
+
+// Person represents a homo sapien instance.
+type Person struct {
```

Comment:
nit: this seems the same for every case, so you could just have the field list as the input, and add the rest in the loop.

Reply:
The example codes is different in second test case.